crates/rspack_plugin_javascript/src/plugin/inner_graph_plugin.rs

```diff
@@ -460,6 +483,8 @@ impl<'a> Visit for InnerGraphPlugin<'a> {
         self.clear_symbol_if_is_top_level();
       }
       _ => {
+        // keep the parens to support case like `/*#__PURE__*/(foo())`
+        let expr = &node.expr;
```

Comment:
Fixed a case where nested 
call expression failed to be recognized as side effect free, for example: `/*#__PURE__*/((foo()))` (previously normalized to a single paren by `swc`)